spec/support/billing_spec_helper.rb

```diff
@@ -52,6 +68,17 @@ def billing_plan_response_body(attributes={})
       }.deep_merge(attributes)
     end
 
+    def billing_free_plan_response_body(attributes={})
+      {
+        "id" => "travis-ci-free-build",
+        "name" => "Startup",
+        "builds" => 0,
```

Comment:
Is this only used by `travis-web` for display? If so, does this want to be 1?

Reply:
yeah,  I need to update that